libpod/container_internal_linux.go

```diff
@@ -482,6 +482,21 @@ func (c *Container) checkpoint(ctx context.Context, options ContainerCheckpointO
 	if c.state.State != ContainerStateRunning {
 		return errors.Wrapf(ErrCtrStateInvalid, "%q is not running, cannot checkpoint", c.state.State)
 	}
+
+	// Create the CRIU log file and label it
+	dumpLog := filepath.Join(c.bundlePath(), "dump.log")
+	logFile, err := os.Create(dumpLog)
```

Comment:
Why not 
```
f, err := os.OpenFile(dumpLog, os.O_RDWR|os.O_CREATE, 0600)	
if err != nil  && os.IsNotExists(err) {
￼			return errors.Wrapf(err, "failed to create CRIU log file %q", dumpLog)
}
r.close()
if err = label.SetFileLabel(dumpLog, c.MountLabel()); err != nil {
		return errors.Wrapf(err, "failed to label CRIU log file %q", dumpLog)
￼}
```

Reply:
You will fail if the file already exists?  If you want to make sure the file is empty you could open it for write.

Reply:
Thanks for the suggestion, using it in the updated version of this PR.